### PR TITLE
ci(kitchen+travis): test with latest Docker images

### PR DESCRIPTION
**Pull request by n-rodriguez**

### CODE REVIEW

File: kitchen.yml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  name: inspec
+  sudo: true
+  # cli, documentation, html, progress, json, json-min, json-rspec, junit
+  reporter:
+    - cli
+
+suites:
+  - name: default
+    excludes:
+      - centos-6-2017-7-py2
```
Comment (myii) [inline kitchen.yml]: This `excludes` can be removed.

File: kitchen.yml [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        top.sls:
+          base:
+            '*':
+              - vim
+      pillars_from_files:
+        vim.sls: test/salt/pillar/vim.sls
+    verifier:
+      inspec_tests:
+        - path: test/integration/default
+  - name: centos6
```
Comment (myii) [inline kitchen.yml]: This entire `centos6` suite can be removed.

File: .travis.yml [hunk 3]
```diff
@@ -1,15 +1,45 @@
...
+    - INSTANCE: default-amazonlinux-2-2019-2-py2
+    # - INSTANCE: default-debian-9-2018-3-py2
+    # - INSTANCE: default-ubuntu-1604-2018-3-py2
+    # - INSTANCE: default-centos-7-2018-3-py2
+    - INSTANCE: default-fedora-29-2018-3-py2
+    - INSTANCE: default-opensuse-leap-15-2018-3-py2
+    # - INSTANCE: default-amazonlinux-2-2018-3-py2
+    # - INSTANCE: default-debian-8-2017-7-py2
+    # - INSTANCE: default-ubuntu-1604-2017-7-py2
+    - INSTANCE: centos6-centos-6-2017-7-py2
```
Comment (myii) [inline .travis.yml]:
```suggestion
    - INSTANCE: default-centos-6-2017-7-py2
```

**Comment by myii** [comment]
@n-rodriguez Note, a similar thing slipped through in https://github.com/saltstack-formulas/timezone-formula/pull/12.

**Comment by n-rodriguez** [comment]
> @n-rodriguez Note, a similar thing slipped through in [saltstack-formulas/timezone-formula#12](https://github.com/saltstack-formulas/timezone-formula/pull/12).

ok

**Comment by n-rodriguez** [comment]
> What do you think is the best way to document this?

pretty good question ^^

**Comment by myii** [comment]
@n-rodriguez I thought you would have a good idea since you've done some conversions since this was added to the `template-formula`.  Would it be best as an inline comment above the suites section?

**Comment by n-rodriguez** [comment]
> Would it be best as an inline comment above the suites section?

yes :+1:

**Comment by n-rodriguez** [comment]
hi there! any news!

**Pull request merged**